src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkRuleContext.java

```diff
@@ -1133,19 +1133,34 @@ private static void checkDeprecated(String newApi, String oldApi, StarlarkSemant
   }
 
   /**
-   * Builds a map: Label -> List of files from the given labels
+   * Builds a map: Label -> List of files from the given labels.
+   * It first looks into the runfiles and then into files. If the label being iterated is an
```

Comment:
Please update this comment to s/runfiles/files-to-run/

Reply:
Addressed in d8176177c9.